packages/database/src/migrations/20210512114642-UpdatePSSSTotalSitesReportedIndicator-modifies-data.js

```diff
@@ -0,0 +1,141 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+const NEW_INDICATOR_CONFIG = {
+  formula:
+    'firstExistingValue(PSSS_Sites_Reported, countrySiteWeeklyResponsesCount, countrySiteDailyResponsesCount, singleSiteWeeklyResponseCount, singleSiteDailyResponseCount)',
+  parameters: [
+    {
+      code: 'countrySiteWeeklyResponsesCount',
+      config: {
+        formula: 'PSSS_Site_Weekly_Survey_Completed',
+        aggregation: [
+          'FINAL_EACH_WEEK',
+          {
+            type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+            config: {
+              dataSourceEntityType: 'facility',
+              aggregationEntityType: 'country',
+            },
+          },
+        ],
+      },
+      builder: 'analyticArithmetic',
+    },
+    {
+      code: 'countrySiteDailyResponsesCount',
+      config: {
+        formula: 'PSSS_Site_Daily_Survey_Completed',
+        aggregation: [
+          'FINAL_EACH_WEEK',
+          {
+            type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
+            config: {
+              dataSourceEntityType: 'facility',
+              aggregationEntityType: 'country',
+            },
+          },
+        ],
+      },
+      builder: 'analyticArithmetic',
+    },
+    {
+      code: 'singleSiteWeeklyResponseCount',
+      config: {
+        formula: 'PSSS_Site_Weekly_Survey_Completed',
+        aggregation: ['FINAL_EACH_WEEK'],
+      },
+      builder: 'analyticArithmetic',
+    },
```

Comment:
```
 PSSS_Site_Daily_Survey_Completed: [
      'FINAL_EACH_WEEK',
      {
        type: 'SUM_PER_PERIOD_PER_ORG_GROUP',
        config: {
          dataSourceEntityType: 'facility',
          aggregationEntityType: 'country',
        },
      },
    ],
```
Not sure If I'm right, but previously to get the number of reported sites at sentinel site level, we use the config above. However, that won't work if we are requesting sentinel sites organisation units (will return nothing). And because of that, WoW Increase and Threshold Crossed indicators won't return anything if requesting sentinel sites organisation units also. So I'm modifying the indicator config here to support the case.